scripts/nanoplen_two_col_plots.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+"""
+Pivot (convert table form long-to-wide format), log2-transform and create plots
```

Comment:
The long to wide are just implementation details that the user does not care. Just describe what the script offers.